### DISCUSSION TITLE
integration menu "ConEmu Here" open up ConEmu, How can I replace it to Cmder?

### DISCUSSION THREAD

**Discussion by rsyqvthv** [cmderdev/cmder#2890] [General]
Once I manually open a Cmder window and then utilize the "ConEmu Here" command from the context menu, the new window combines seamlessly, adorned with Cmder's icon. 

I aspire to achieve a harmonious interface but remain unsure about the process to configure ConEmu, so that invoking it from the context menu integrates with Cmder.

**Comment by daxgames** [comment]
1. You launch Cmder
2. You use context menu `Conemu Here` to launch into a specific folder.
3. You know have 2 separate windows instead of one.

Is this correct?

Why do you have a `Conemu Here` context menu instead of a `Cmder Here` context menu?

Get a `Cmder Here` context menu:

```
Cmder /register user
```

Does this solve the issue?

**Reply by rsyqvthv** [reply]
Thanks @daxgames , I tried `cmder /regsiter user`, but seems it doesn't work like what I expect, it can only open cmder when there isn't any exist instance. if there is cmder instance, the "cmder here" only minimized exist cmder window, and no new tab will create. 

but the "ConEmu here" works, it opens 2nd tab in cmder.

as I checked what `cmder /register user` did in regedit, it shows me following in the `command` key: 

`"D:\path\Cmder.exe" "%V"`

not sure if it's correct? 

I am using the last version of cmder_mini, in windows 10 1809.
